File: 15701.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
	int N; cin >> N;

	int cnt = 1;
	for (int i = 2; i <= sqrt(N); i++)
		if (N % i == 0) cnt++;

	cnt *= 2;
	if ((int)sqrt(N) * (int)sqrt(N) == N) cnt--;

	cout << cnt << endl;
}

File: 15701.py
from math import sqrt

N = int(input())

cnt = 1
for i in range(2, int(sqrt(N)) + 1):
    if N % i == 0: cnt += 1

cnt *= 2
if int(sqrt(N)) ** 2 == N:
    cnt -= 1

print(cnt)
